packages/headless/src/product-listing.index.ts

```diff
@@ -170,3 +170,10 @@ export type {
   ContextValue,
 } from './controllers/product-listing/context/headless-product-listing-context';
 export {buildContext} from './controllers/product-listing/context/headless-product-listing-context';
+
+export type {
+  InteractiveResultOptions,
+  InteractiveResultProps,
+  InteractiveResult,
+} from './controllers/product-listing/result-list/headless-product-listing-interactive-result';
+export {buildInteractiveResult} from './controllers/product-listing/result-list/headless-product-listing-interactive-result';
```

Comment:
If we want to include this controller in the docs, we'll need to add it to [this array](https://github.com/coveo/ui-kit/blob/master/packages/headless/doc-parser/use-cases/product-listing.ts#L5)

Reply:
Thanks !

Reply:
Yea, if CI is erroring, it's possible that documentation is missing on the controller.
If this is the case, it should be possible to recreate the error locally by running `npm run build`.